packages/loader/package.json

```diff
@@ -49,11 +49,11 @@
     "loader-utils": "^2.0.0"
   },
   "devDependencies": {
-    "memory-fs": "^0.5.0",
     "react": "^17.0.0",
     "react-dom": "^17.0.0",
     "remark-slug": "^6.0.0",
-    "webpack": "^4.0.0"
+    "webpack": "^5.51.1"
```

Comment:
Upgrade to webpack 5 for https://github.com/webpack/loader-runner/pull/40, although it's still missing https://github.com/webpack/webpack/pull/14077.